# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1715705273 -7200
# Parent  ef01a739d7c07eebe25047eb6ee75e900e079d8b
Misc: Default unified folder to 'no offline search' to avoid unexpected performance hits.

diff --git a/mail/modules/SmartMailboxUtils.sys.mjs b/mail/modules/SmartMailboxUtils.sys.mjs
--- a/mail/modules/SmartMailboxUtils.sys.mjs
+++ b/mail/modules/SmartMailboxUtils.sys.mjs
@@ -148,9 +148,9 @@ class SmartMailbox {
           "searchFolderUri",
           searchFolders.map(f => f.URI).join("|")
         );
         folderInfo.setUint32Property("searchFolderFlag", folderType.flag);
-        folderInfo.setBooleanProperty("searchOnline", true);
+        folderInfo.setBooleanProperty("searchOnline", false);
         msgDatabase.summaryValid = true;
         msgDatabase.close(true);
 
         this.#rootFolder.notifyFolderAdded(folder);
